bugalink-frontend/constants/nextRoutes.ts

```diff
@@ -10,7 +10,7 @@ const NEXT_ROUTES = {
   MY_ROUTINES: `/routines`,
   RIDE_HISTORY: '/history',
   RIDE_DETAILS_ONE: (rideId, requested = false) =>
-    `/ride/${rideId}/detailsOne?requested=${requested}`,
+    `/ride/${rideId}/details?requested=${requested}`,
   RIDE_DETAILS_TWO: (rideId) => `/ride/${rideId}/detailsTwo`,
```

Comment:
This should be deleted

Reply:
All references of details two has been deleted in a new commit